.eslintrc

```diff
@@ -0,0 +1,65 @@
+
+{
+  // Extend the AirBnb lint config
+  "extends": "airbnb",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "ecmaFeatures": {
+      "globalReturn": true,
+      "impliedStrict": true,
+      "jsx": true
+    },
+    "sourceType": "module",
+  },
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true,
+    "jquery": true,
+    // Optional Enables
+    "webextensions": false, // Enable if using Web Extensions
+    // Optional Testing Frameworks
+    "jasmine": false, // Enable if using Jasmine testing framework
+    "protractor": false, // Enable if using Protractor testing framework
+    "mocha": false // Enable if using Mocha testing framework
+  },
+  "globals": {
+    "jQuery": true,
+    "angular": false, // Enable if using Angular
+  },
+  // Do NOT change these rules
+  "rules": {
+    "indent": [2, 2, {"SwitchCase": 1}],
+    "max-len": [2, 80, 4, {
+      "ignoreComments": true,
+      "ignoreUrls": true,
```

Comment:
this required updating eslint to `3.8.1` which produced the following:

```
Joshs-MacBook-Pro:simplechart joshkadis$ npm update eslint
- esutils@1.1.6 node_modules/doctrine/node_modules/esutils
- bluebird@3.4.1 node_modules/table/node_modules/bluebird
- read-json-sync@1.1.1 node_modules/read-json-sync
- tv4@1.2.7 node_modules/tv4
- xregexp@3.1.1 node_modules/xregexp
simplechart-react@1.0.0 /Users/joshkadis/Repos/simplechart
├── UNMET PEER DEPENDENCY eslint@3.8.1
├── UNMET PEER DEPENDENCY inline-style-prefixer@^2.0.1
├── UNMET PEER DEPENDENCY react@15.2.1
└── UNMET PEER DEPENDENCY react-dom@15.2.1

npm WARN better-react-spinkit@2.0.0-3 requires a peer of inline-style-prefixer@^2.0.1 but none was installed.
npm WARN eslint-config-airbnb@6.0.2 requires a peer of eslint@^2.2.0 but none was installed.
npm WARN react-colorpickr@4.2.0 requires a peer of react@0.14.x but none was installed.
npm WARN react-colorpickr@4.2.0 requires a peer of react-dom@0.14.x but none was installed.
npm WARN react-fade@1.1.1 requires a peer of inline-style-prefixer@^2.0.1 but none was installed.
npm WARN simplechart-react@1.0.0 No license field.
npm ERR! code 1
```
Which I'm ignoring for now since linting seems to work. Would appreciate an audit of the package deps at some point.